client/js/components/procedure/admin/InstitutionTagManagement/TagList.vue

```diff
@@ -222,27 +224,31 @@ export default {
     },
 
     transformTagsAndCategories () {
-      return Object.values(this.institutionTagCategories).map(category => {
-        const { attributes, id, type } = category
-        const tags = category.relationships?.tags?.data.length > 0 ? category.relationships.tags.list() : []
-
-        return {
-          id,
-          name: attributes.name,
-          children: Object.values(tags).map(tag => {
-            const { id, attributes, type } = tag
-
-            return {
-              id,
-              categoryId: category.id,
-              isUsed: attributes.isUsed,
-              name: attributes.name,
-              type
-            }
-          }),
-          type
-        }
-      })
+      return Object.values(this.institutionTagCategories)
+        .sort((a, b) => new Date(b.attributes.creationDate) - new Date(a.attributes.creationDate))
+        .map(category => {
+          const { attributes, id, type } = category
+          const tags = category.relationships?.tags?.data.length > 0 ? category.relationships.tags.list() : []
+
+          return {
+            id,
+            name: attributes.name,
+            children: Object.values(tags)
+              .sort((a, b) => new Date(b.attributes.creationDate) - new Date(a.attributes.creationDate))
```

Comment:
Do we also want to sort the tags by creationDate? I think alphabetic (or better, manual sorting) would be better here.

Reply:
I think tags should follow the same logic as categories, isn't it confusing otherwise? Why do you think alphabetic order might be better?
Since this is just an improvement on the side I didn't want to get into manual sorting, and as long as we don't have that sorting by `creationDate` gives at least some kind of control to the user.

Reply:
ok i think we can go with creationDate for now, but ultimately this forces users to understand the sorting mechanism by trial & error, and then creating the needed tags in reverse order of the desired "end order". 

to do better, we have to either add a setting per category (which is kind of overkill i guess), or implement a manual sorting mechanism for tags, which may add overhead for users who then are forced to sort large lists alphabetically, but maybe some tags are better sorted manually (like "Verfahrensschritte", which need to appear in a custom order). so maybe a 3rd improvement would be to offer a "sort alphabetically" action on each category within the admin interface.

Reply:
This
> this forces users to understand the sorting mechanism by trial & error, and then creating the needed tags in reverse order of the desired "end order"

is a really good point, and it is also true for categories. Should the sort order be descending by default, so at least you don't have to reverse the order as a user?

The manual sorting (either by drag+drop or via button/flyout) I think we should discuss with the product owner and create a user story.

Reply:
>Should the sort order be descending by default, so at least you don't have to reverse the order as a user?

Maybe this is a good idea, yes.

Reply:
Done in 0b7f97a09a3229982613ffbc29f9ee92effe5111